tracking/src/SessionManager.js

```diff
@@ -0,0 +1,91 @@
+
+const SESSION_TIMEOUT = 1800000; 
+const DAY_IN_MS = 86400000; 
+const KEY_UID = "va-uid"; 
+const KEY_SID = "va-key-sid";
+const KEY_SID_TIMESTAMP = "va-key-sid-timestamp"; 
+
+class SessionManager {
+    constructor(uid) {
+
+        this.uid = (uid === undefined) ? this.getUID() : uid; 
+        this.sid = this.getSID(); 
+        this.sessionTimestamp =  this.getSessionTimestamp(); 
+        console.log("sessionManager init, uid: ", this.uid);
+    }
+
+    isSameDay(t1, t2) {
```

Comment:
can add in the timezone check: https://github.com/googleanalytics/autotrack/blob/master/lib/session.js#L192

Reply:
changed